app/dashboard/router.py

```diff
@@ -171,10 +182,10 @@ def get_queryset(self):
         queryset = queryset.distinct()
 
         # offset / limit
-        limit = self.request.query_params.get('limit', None)
-        offset = self.request.query_params.get('offset', 0)
-        if limit:
-            queryset = queryset[int(offset):int(limit)]
+    #    limit = min(100,int(self.request.query_params.get('limit', 100)))
+    #    offset = self.request.query_params.get('offset', 0)
+    #    if limit:
+    #    queryset = queryset[int(offset):2]
```

Comment:
^ I'm assuming the commented code is going to used (if not we could do away with it )